common/option-model.cpp

```diff
@@ -259,7 +260,7 @@ bool option_model::draw_combobox( notifications_model & model,
     {
         int tmp_selected = selected;
         float tmp_value = value;
-        ImGui::PushItemWidth( 105.f );
```

Comment:
The line was removed because we drew Combo to the end of the line here [PushItemWidth(-1.f)](https://github.com/IntelRealSense/librealsense/pull/12588/files#diff-593e9b0b372f0f928d63fba53e5d24e02f3c807182b1290219f3bd0f4c3fa1ecL244) so no need set width. 

More of all I found that we perform PopItemWidth() and not PopItemWidth(2). Is this correct?

Reply:
I think that if the push count != pop count we get an asseration in debug mode

Reply:
This force the combo box width, what happens if we remove it?
Please share internally a release flow build so I can play with the viewer

Reply:
> I think that if the push count != pop count we get an asseration in debug mode

Yes, you are right, I encountered the described behavior but in another place/code. But in our case, it worked before and after removing... That is very weird. I will check it.

Reply:
I checked and found that we get an assertion only when pop > count.
There was pop < count. So it worked.

Also, I added `auto stack_size = window->DC.ItemWidthStack.size();` in `void ImGui::PopItemWidth()` in `imgui.cpp`,

> void ImGui::PopItemWidth()
{
    ImGuiWindow* window = GetCurrentWindow();
    window->DC.ItemWidthStack.pop_back();
    window->DC.ItemWidth = window->DC.ItemWidthStack.empty() ? window->ItemWidthDefault : window->DC.ItemWidthStack.back();
    auto stack_size = window->DC.ItemWidthStack.size();
}

And found that stack_size = 1 in the end of **our** function but after removing unnecessary "push" it 0 as expected.